.github/workflows/build-release.yml

```diff
@@ -15,7 +15,9 @@ jobs:
         with:
           version: latest
       - name: install plugin
-        run: packer plugins install github.com/digitalocean/digitalocean >=1.1.1
+        env:
+          FF_VERSION: ${{ github.ref_name }}
```

Comment:
This will have a `v` at the start which is different to the current droplet name which is just the numbers. Is that okay?

Reply:
The script `build.sh` will strip the leading v now